test/NuGet.Clients.Tests/NuGet.CommandLine.Test/NuGetPackCommandTest.cs

```diff
@@ -5578,6 +5578,156 @@ public void PackCommand_PackageFromNuspecWithFrameworkReferences_WithDuplicateEn
             }
         }
 
+        [Fact]
+        public void PackCommand_PackIcon_HappyPath_Succeed()
+        {
+            var iconTestSourceDir = new IconTestSourceDirectory("icon.jpg", "icon.jpg", 6);
+            TestPackIcon(iconTestSourceDir);
+        }
+
+        [Fact]
+        public void PackCommand_PackIcon_ImplicitFile_Succeed()
+        {
+            var fileEntries = new List<string>();
+            var files = new List<Tuple<string, int>>();
+            var _ = Path.DirectorySeparatorChar;
```

Comment:
Don't use _ as a variable name. 
`_` is used for discard.

Reply:
Changed

Reply:
For reference: https://docs.microsoft.com/en-us/dotnet/csharp/discards